app/src/cc/arduino/packages/formatter/AStyle.java

```diff
@@ -76,15 +76,27 @@ public void init(Editor editor) {
 
   @Override
   public void run() {
-    String originalText = editor.getCurrentTab().getText();
+    
+    String originalText = editor.getCurrentTab().getSelectedText();
+    boolean selection = true;
+    
+    // If no selection use all file
+    if(originalText == null || originalText.isEmpty()) {
```

Comment:
I think there should be a space after `if` here, looks like that is the convention used here.

Reply:
Done...